packages/modules/mssqlserver/src/mssqlserver-container.ts

```diff
@@ -0,0 +1,123 @@
+import { AbstractStartedContainer, GenericContainer, StartedTestContainer } from "testcontainers";
+
+const MSSQL_PORT = 1433;
+
+type ProductKey = `${string}-${string}-${string}-${string}-${string}`;
+
+type Edition = "Evaluation" | "Developer" | "Express" | "Web" | "Standard" | "Enterprise" | ProductKey;
+
+const PASSWORD_CATEGORY_VALIDATION_PATTERNS = [/[A-Z]+/, /[a-z]+/, /[0-9]+/, /[@$!%*?&]+/];
+
+export class MSSQLServerContainer extends GenericContainer {
+  private database = "master";
+  private username = "SA";
+  private password = "Passw0rd";
+  private edition: Edition = "Developer";
+  private acceptEula = "N";
+
+  constructor(image = "mcr.microsoft.com/mssql/server:2022-latest") {
+    super(image);
+  }
+
+  public acceptLicense(): this {
+    this.acceptEula = "Y";
+    return this;
+  }
+
+  public withDatabase(database: string): this {
+    this.database = database;
+    return this;
+  }
```

Comment:
AFAIK, you can not set the database name in MSSQL and due to there is no test covering the case I wonder if that works, nowadays.

Reply:
I left this in as a convenience method for the generating the connection string should the user want to seed the instance during initialization. I can also add a test/example for seeding the database with SQL files and bash scripts but I was worried about adding too many files. Let me know how I should proceed.